src/apify_client/clients/resource_clients/actor.py

```diff
@@ -0,0 +1,317 @@
+        webhooks: Optional[List[Dict]] = None,
+    ) -> Dict:
+        """Start the actor and immediately return the Run object.
+
+        https://docs.apify.com/api/v2#/reference/actors/run-collection/run-actor
+
+        Args:
+            run_input (Any, optional): The input to pass to the actor run.
+            content_type (str, optional): The content type of the input.
+            build (str, optional): Specifies the actor build to run. It can be either a build tag or build number.
+                                   By default, the run uses the build specified in the default run configuration for the actor (typically latest).
+            memory_mbytes (int, optional): Memory limit for the run, in megabytes.
+                                           By default, the run uses a memory limit specified in the default run configuration for the actor.
+            timeout_secs (int, optional): Optional timeout for the run, in seconds.
+                                          By default, the run uses timeout specified in the default run configuration for the actor.
+            wait_for_finish (int, optional): The maximum number of seconds the server waits for the run to finish.
+                                               By default, it is 0, the maximum value is 300.
+            webhooks (list of dict, optional): Optional ad-hoc webhooks (https://docs.apify.com/webhooks/ad-hoc-webhooks)
+                                               associated with the actor run which can be used to receive a notification,
+                                               e.g. when the actor finished or failed.
+                                               If you already have a webhook set up for the actor or task, you do not have to add it again here.
+                                               Each webhook is represented by a dictionary containing these items:
+                                               * ``event_types``: list of ``WebhookEventType`` values which trigger the webhook
+                                               * ``request_url``: URL to which to send the webhook HTTP request
+                                               * ``payload_template`` (optional): Optional template for the request payload
+
+        Returns:
+            dict: The run object
+        """
+        run_input, content_type = _encode_key_value_store_record_value(run_input, content_type)
+
+        request_params = self._params(
+            build=build,
+            memory=memory_mbytes,
+            timeout=timeout_secs,
+            waitForFinish=wait_for_finish,
+            webhooks=_encode_webhook_list_to_base64(webhooks) if webhooks is not None else None,
+        )
+
+        response = self.http_client.call(
+            url=self._url('runs'),
+            method='POST',
+            headers={'content-type': content_type},
+            data=run_input,
+            params=request_params,
+        )
```

Comment:
Btw, just an idea, but can't we create a function `post()` on `http_client`? Then, instead of `.call(method='POST')`, we would be able to call `.post()` directly. Would make the code more readable imho

Reply:
Then we'd also have to add `.get()`, `.put()` and `.delete()`, to be consistent, and all of these methods would be just wrapper methods simply passing all it's parameters down plus adding one extra. It would be ~80 extra lines of mostly duplicated code with little added benefit, I don't think it's worth it.